package.json

```diff
@@ -9,6 +9,7 @@
     "build-localized": "ng build --configuration=production,localized",
     "test": "ng test",
     "test-ci": "ng test --code-coverage --karmaConfig=build/karma-ci.conf.js ",
+    "test-cov": "ng test --code-coverage --karmaConfig=karma.conf.js ",
```

Comment:
```suggestion
```
I don't think this is needed (the `package.json` already has so many scripts). If you want to run tests with coverage locally you can also use
`npm test -- --code-coverage`, everything after the first `--` is passed to the `ng test` command.